internal/quantile/aggregate.go

```diff
@@ -74,36 +74,43 @@ func (e *E2eProcessingLatencyAggregate) Less(i, j int) bool {
 	return e.Percentiles[i]["percentile"] > e.Percentiles[j]["percentile"]
 }
 
+// Add merges two E2eProcessingLatencyAggregate by averaging the percentiles
+//
+// It is side-effect free and does not modify the receiver or the parameter
 func (e *E2eProcessingLatencyAggregate) Add(e2 *E2eProcessingLatencyAggregate, N int) *E2eProcessingLatencyAggregate {
 	if e == nil {
-		*e = *e2
```

Comment:
Sorry for this bug. Probably what I meant to do is

``` go
if e == nil {
  e = &E2eProcessingLatencyAggregate{}
  *e = *e2
} else {
  ...
```

But I also didn't realize that this function was not supposed to modify the receiver. It looked like a `math/big`-style API.